app/javascript/components/student/solution-summary/Header.tsx

```diff
@@ -2,18 +2,34 @@ import React from 'react'
 import { GraphicalIcon, Icon } from '../../common'
 import { Iteration, IterationStatus } from '../../types'
 import pluralize from 'pluralize'
+import { ExerciseType } from '../SolutionSummary'
 
 export type SolutionSummaryLinks = {
   testsPassLocallyArticle: string
 }
 
+const TutorialHeader = () => {
+  return (
+    <header>
+      <div className="info">
+        <h2>Your solution looks good!</h2>
+        <p>
+          <strong>Good job.</strong> Your solution to &quot;Hello, World!&quot;
```

Comment:
It is highly unlikely, but what if we decide to change the title of the exercise (or use a different one)? Do you have the exercise's title as data somewhere? If so, you could consider using that here.

Reply:
```suggestion
          {/* TODO: Use exercise.title anywhere where we have Hello, World */}
          <strong>Good job.</strong> Your solution to &quot;Hello, World!&quot;
```